redhat-operators/openshift-pipelines-operator/openshift-pipelines-operator-tekton_v1alpha1_install_crd.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: installs.tekton.dev
+spec:
+  group: tekton.dev
+  names:
+    kind: Install
+    listKind: InstallList
+    plural: installs
+    singular: install
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+        status:
+          properties:
+            resources:
+              description: The resources applied
+              items:
+                type: string
+              type: array
+            version:
+              type: string
+          required:
```

Comment:
Wondering, what could happen if it's not marked as required? I was under impression `required` apply constraint during resource creation/update, right?

Reply:
Right now we don't have any spec defined.
That required is in status which is handled by the Operator so the user will not have to worry about this required statement

Reply:
Right now we don't have any spec defined.
That required is in status which is handled by the Operator so the user will not have to worry about this required statement.